documentation/faq/70-packages.md

```diff
@@ -2,6 +2,19 @@
 question: How do I fix the error I'm getting trying to include a package?
 ---
 
-Most of these issues come from Vite trying to deal with non-ESM libraries. You may find helpful examples in [the Vite issue tracker](https://github.com/vitejs/vite/issues). The most common solutions would be to try moving the package between `dependencies` and `devDependencies` or trying to `include` or `exclude` it in `optimizeDeps`. Packages which use `exports` instead of `module.exports` are currently failing due to a [known Vite issue](https://github.com/vitejs/vite/issues/2579). You should also consider asking the library author to distribute an ESM version of their package or even converting the source for the package entirely to ESM.
+If youre seeing errors like:
+
+* `[vite] Error when evaluating SSR module /node_modules/....`
+* `ReferenceError: require is not defined`
+
+Most likely they are coming from Vite trying to deal with non-ESM libraries. You may find helpful examples in [the Vite issue tracker](https://github.com/vitejs/vite/issues). 
+
+The most common solutions:
+
+1. Try moving the package between `dependencies` and `devDependencies`
+2. Try to `include` or `exclude` it in `optimizeDeps`
+3. Try to find an ESM build of the package, if available. If not available, you should also consider asking the library author to distribute an ESM version of their package or even converting the source for the package entirely to ESM.
```

Comment:
"Try to find an ESM build of the package, if available." - this sounds a bit confusing to me. The ESM build is almost always in the package in question or not available. The only thing I can think of is the redaxios suggestion mentioned later on, but it's really uncommon that someone would have made an ESM fork of a package, so I wouldn't suggest people go looking for forks of the package

Reply:
I removed all that about finding an ESM package